tests/console/supportutils.pm

```diff
@@ -23,8 +23,9 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
+    my $options = get_var('SUPPORTCOFIG_OPTIONS') // 'BOOT,BTRFS,DAEMONS,CRASH,MEM,NET,PROC,SYSCONFIG';
```

Comment:
This test was written as a replacement for manual regression testing and therefor limiting it's scope means lowering the test coverage.
I don't mind anyone not running full supportconfig but I'm not a fan of forcing everyone else to add variables they don't need. If you manage to do it in a way where adding these options is optional, I'm fine with it. At this point, however, it will brake our runs.

Reply:
Yes, here the setting of SUPPORTCOFIG_OPTIONS is optional, and if you don't set it, we will use 'BOOT,BTRFS,DAEMONS,CRASH,MEM,NET,PROC,SYSCONFIG' as default value. Thanks.

Reply:
Yes, but that is not full set of possibilities that supportconfig offers.

Reply:
I understand what you thinking, but now full set will cost us 1600s. So maybe you'd like to run full set as default, while others set the SUPPORTCOFIG_OPTIONS for special requirement?

Reply:
It will be better to define in `SUPPORTCOFIG_OPTIONS` the whole option with arguments.
Default value will be nothing, so run default as until now. set timeout to 2000 as it can take long, depends on the load of openQA
SUPPORTCOFIG_OPTIONS can be set as anybody requires in test
```
-    assert_script_run "supportconfig -t . -B test", 800;
+    my $options = get_var('SUPPORTCOFIG_OPTIONS', '');
+    assert_script_run "supportconfig $options -t . -B test", 2000;
```

Reply:
default http://dzedro.suse.cz/tests/17036#step/supportutils/15
`SUPPORTCOFIG_OPTIONS ='-i BOOT'` http://dzedro.suse.cz/tests/17037#settings

Reply:
Thanks, that was exactly what I was thinking. Not using '-i' at all if it's not needed. Should be the most future/fail-proof way to run it in any way we want.

Reply:
Thanks, I have updated the code. I will run verification log later since the clone job not work for openQA now.

Reply:
Just run the verification logs, wait:
SUPPORTCOFIG_OPTIONS='-i BOOT,BTRFS,DAEMONS':  http://openqa.nue.suse.com/tests/5145822#
default:  http://openqa.nue.suse.com/tests/5145823#

Reply:
The verification log is ok now.  
SUPPORTCOFIG_OPTIONS='-i BOOT,BTRFS,DAEMONS': 
 http://openqa.nue.suse.com/tests/5145822#step/supportutils/5
default:  http://openqa.nue.suse.com/tests/5145823#step/supportutils/5